src/core_ocean/mode_init/mpas_ocn_init_tidal_boundary.F

```diff
@@ -363,14 +435,19 @@ subroutine ocn_init_setup_tidal_boundary(domain, iErr)!{{{
       ! Set tidal boundary mask
       do iCell = 1, nCellsSolve
         tidalInputMask(iCell) = 0.0_RKIND
-        if (yCell(iCell) > (25.0e3 - dcEdgeMinGlobal/2.0_RKIND)) then
+        if (yCell(iCell) > (yMax - dcEdgeMinGlobal/2.0_RKIND)) then ! Zhendong March-05-2020
```

Comment:
Nice change to generalize `yMax`.  No need for ` ! Zhendong March-05-2020` as it will be stored in git repo and can be found via a `git blame` on this line.

Reply:
Will clean it up.